neqo-http3/src/connection.rs

```diff
@@ -280,9 +230,18 @@ impl Http3Connection {
         });
     }
 
+    fn create_qpack_streams(&mut self) -> Res<()> {
+        qdebug!([self] "create_qpack_streams.");
```

Comment:
Fixed, but there are other spots :smiley: can you do a quick grep for qdebugs and just convert to qtrace if they just print the function name? (or maybe consider removing them, since maybe the value they provide isn't high enough to justify if they only print the function name...)